gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# Improved UI for matches

### Description

Improved the UI for the Matches Page.

### Testing Directions

```
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
```

Make 2 users or more interested/going to an event. Make a match between 2 users, they should be appearing on Matches tab. Now the matches.html looks something similar to home.html. Also added Message, User Profile link in matches.

### Miscellaneous Comments


### Checklist
